benchmarks/diffusion_2D/mpi_gpu/CMakeLists.txt

```diff
@@ -74,6 +74,28 @@ foreach(test_tuple ${tests})
 
   add_executable(${target} ${sources})
 
+  if("${backend}" STREQUAL "USE_CUDA")
+
+    # based on Lassen system from build_and_bench (1 nodes).
+    # sundials_add_benchmark(${target} ${target} diffusion_2D
```

Comment:
Should this be removed or uncommented?

Reply:
This should be left uncommented, it's for adding the Diffusion 2D benchmark on Lassen, which currently errors out.

Reply:
@yu-nix Can you copy the error to here so we can investigate it further?

Reply:
This is the output if you run `arkode_diffusion_2D_mpicuda`

```

 Problem options:
 --------------------------------- 
  nprocs         = 4
  npx            = 2
  npy            = 2
 --------------------------------- 
  kx             = 1
  ky             = 1
  forcing        = 1
  tf             = 1
  xu             = 1
  yu             = 1
  nx             = 32
  ny             = 32
  nxl (proc 0)   = 16
  nyl (proc 0)   = 16
  dx             = 0.0322581
  dy             = 0.0322581
 --------------------------------- 

 Integrator options:
 --------------------------------- 
 rtol        = 1e-05
 atol        = 1e-10
 hfixed      = 0
 order       = 3
 controller  = 0
 max steps   = 0
 linear RHS  = 1
 diagnostics = 0
 --------------------------------- 

 Linear solver options:
 --------------------------------- 
 LS       = cg
 precond  = 1
 LS info  = 0
 LS iters = 20
 msbp     = 0
 epslin   = 0
 --------------------------------- 

 Output options:
 --------------------------------- 
 output level = 1
 nout         = 20
 --------------------------------- 

          t                     ||u||_rms                max error      
 -----------------------------------------------------------------------
 0.000000000000000e+00    3.632812500000001e-01    0.000000000000000e+00
[lassen13:151047] *** Process received signal ***
[lassen13:151047] Signal: Segmentation fault (11)
[lassen13:151047] Signal code: Invalid permissions (2)
[lassen13:151047] Failing at address: 0x200053400200
[lassen13:151050] *** Process received signal ***
[lassen13:151050] Signal: Segmentation fault (11)
[lassen13:151050] Signal code: Invalid permissions (2)
[lassen13:151050] Failing at address: 0x200053400200
[lassen13:151048] *** Process received signal ***
[lassen13:151048] Signal: Segmentation fault (11)
[lassen13:151048] Signal code: Invalid permissions (2)
[lassen13:151048] Failing at address: 0x200053400200
[lassen13:151049] *** Process received signal ***
[lassen13:151049] Signal: Segmentation fault (11)
[lassen13:151049] Signal code: Invalid permissions (2)
[lassen13:151049] Failing at address: 0x200053400200
<< Rank 3: Generating lwcore_cpu.5074604_10.3 on lassen13 Wed Jul 26 15:22:26 PDT 2023 (LLNL_COREDUMP_FORMAT_CPU=lwcore) >>
<< Rank 2: Generating lwcore_cpu.5074604_10.2 on lassen13 Wed Jul 26 15:22:26 PDT 2023 (LLNL_COREDUMP_FORMAT_CPU=lwcore) >>
<< Rank 0: Generating lwcore_cpu.5074604_10.0 on lassen13 Wed Jul 26 15:22:26 PDT 2023 (LLNL_COREDUMP_FORMAT_CPU=lwcore) >>
<< Rank 1: Generating lwcore_cpu.5074604_10.1 on lassen13 Wed Jul 26 15:22:26 PDT 2023 (LLNL_COREDUMP_FORMAT_CPU=lwcore) >>
<< Rank 1:  Generated lwcore_cpu.5074604_10.1 on lassen13 Wed Jul 26 15:22:27 PDT 2023 in 1 secs >>
<< Rank 0:  Generated lwcore_cpu.5074604_10.0 on lassen13 Wed Jul 26 15:22:27 PDT 2023 in 1 secs >>
<< Rank 3:  Generated lwcore_cpu.5074604_10.3 on lassen13 Wed Jul 26 15:22:27 PDT 2023 in 1 secs >>
<< Rank 2:  Generated lwcore_cpu.5074604_10.2 on lassen13 Wed Jul 26 15:22:27 PDT 2023 in 1 secs >>
<< Rank 2: Waiting 60 secs before aborting task on lassen13 Wed Jul 26 15:22:27 PDT 2023 (LLNL_COREDUMP_WAIT_FOR_OTHERS=60) >>
<< Rank 3: Waiting 60 secs before aborting task on lassen13 Wed Jul 26 15:22:27 PDT 2023 (LLNL_COREDUMP_WAIT_FOR_OTHERS=60) >>
<< Rank 0: Waiting 60 secs before aborting task on lassen13 Wed Jul 26 15:22:27 PDT 2023 (LLNL_COREDUMP_WAIT_FOR_OTHERS=60) >>
<< Rank 1: Waiting 60 secs before aborting task on lassen13 Wed Jul 26 15:22:27 PDT 2023 (LLNL_COREDUMP_WAIT_FOR_OTHERS=60) >>
<< Rank 2:  Waited 60 secs -> now aborting task on lassen13 Wed Jul 26 15:23:27 PDT 2023 (LLNL_COREDUMP_KILL=task) >>
<< Rank 3:  Waited 60 secs -> now aborting task on lassen13 Wed Jul 26 15:23:27 PDT 2023 (LLNL_COREDUMP_KILL=task) >>
<< Rank 0:  Waited 60 secs -> now aborting task on lassen13 Wed Jul 26 15:23:27 PDT 2023 (LLNL_COREDUMP_KILL=task) >>
<< Rank 1:  Waited 60 secs -> now aborting task on lassen13 Wed Jul 26 15:23:27 PDT 2023 (LLNL_COREDUMP_KILL=task) >>
[lassen13:151049] -----------------------
[lassen13:151049] -----------------------
[lassen13:151050] -----------------------
[lassen13:151050] -----------------------
[lassen13:151049] *** End of error message ***
[lassen13:151050] *** End of error message ***
[lassen13:151048] -----------------------
[lassen13:151048] -----------------------
[lassen13:151047] -----------------------
[lassen13:151047] -----------------------
[lassen13:151048] *** End of error message ***
[lassen13:151047] *** End of error message ***
ERROR:  One or more process (first noticed rank 3) terminated with signal 11

```

Reply:
I don't recall what the issue was with spectrum-mpi and gpu-aware MPI but maybe it's been fixed and adding something like `--smpiargs="-gpu"` to the run command will address the error